libcontainer/cgroups/fs/apply_raw.go

```diff
@@ -179,11 +180,24 @@ func (m *Manager) GetStats() (*cgroups.Stats, error) {
 }
 
 func (m *Manager) Set(container *configs.Config) error {
-	for name, path := range m.Paths {
-		sys, err := subsystems.Get(name)
-		if err == errSubsystemDoesNotExist || !cgroups.PathExists(path) {
+	for _, sys := range subsystems {
+		// We can't set this here, because after being applied, memcg doesn't
+		// allow a non-empty cgroup from having its limits changed.
+		if sys.Name() == "memory" {
```

Comment:
No, this hasn't changed (see 39279b1762f81e0c2d7ebf7f56bf4db23b212dda), it's a specific issue with kernel memory limiting. This is the case for `kmemcg` limits. Since the `kmemcg` limits are integrated into the memory cgroup, I couldn't think of a "nice" way of only setting half of the options.